src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -238,6 +237,58 @@ Similarly, the OPTIONAL [`rec-<label>`](../99-appendices/09-entities.md#rec)
 key/value can be used to distinguish
 different reconstruction algorithms (for example ones using motion correction).
 
+Structural MR images whose intensity is represented in a non-arbitrary scale
+constitute parametric maps. Currently supported parametric maps include:
+
+| **Name**                                     | `suffix`  | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+|----------------------------------------------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Longitudinal relaxation time map             | T1map     | In seconds (s). T1 maps are REQUIRED to use this suffix regardless of the method used to generate them. See [this interactive book on T1 mapping](https://qmrlab.org/t1_book/intro) for further reading on T1-mapping.                                                                                                                                                                                                                                                                                            |
```

Comment:
@effigies as one map can be generated by N different methods, that warning is intended to prevent N --> N creative namings (e.g. RAGET1map, DESPOT1map etc.). Failure to implement this would be creating outputs with invalid suffixes.

In the appendix, I will make a table to list which qMRI maps can be generated from which supported file collections.